micrometer-commons/src/main/java/io/micrometer/common/KeyValues.java

```diff
@@ -409,7 +410,8 @@ public static KeyValues of(@Nullable String... keyValues) {
         return toKeyValues(keyValueArray);
     }
 
-    private static boolean blankVarargs(@Nullable Object[] args) {
+    @Contract("null -> true")
```

Comment:
Without this `@Contract`, NullAway fails to recognize that a Nullable parameter has been null-checked and is safe to pass as a parameter to NonNullable marked methods.